modules/cli/src/main/scala/scala/cli/commands/package0/Package.scala

```diff
@@ -268,9 +268,16 @@ object Package extends ScalaCommand[PackageOptions] with BuildCommandHelpers {
         case _ if Properties.isWin                              => "app.bat"
         case _                                                  => "app"
       }
+      val output = outputOpt.map {
+        case path
+            if packageType == PackageType.GraalVMNativeImage
+            && Properties.isWin && !path.endsWith(".exe") =>
+          s"$path.exe" // graalvm-native-image requires .exe extension on Windows
```

Comment:
maybe log a warning that `.exe` is necessary for the output to run properly, so we add it automatically here.
that'd also remove the need for a code comment.